pkgs/data/themes/levuaska-gtk-theme/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, stdenv, fetchFromGitHub, autoreconfHook, gtk_engines }:
+
+stdenv.mkDerivation {
```

Comment:
Themes don't *usually* need the full `stdenv`, as sometimes compiling with `cc` isn't needed. Is this the case as well? Please check building with `stdenvNoCC` instead.

Reply:
stdenvNoCC does indeed work. I will amend my commit to incorporate this change. Thank you!